docs/cep/query-guide/query.md

```diff
@@ -25,9 +25,9 @@ The following parameters are used to configure a stream definition.
 | Parameter  | Description |
 |----------------|-------------|
 | `query name`   | The name of the query. Since naming the query (i.e the `@name('<query name>')` annotation) is optional, when the name is not provided Stream assign a system generated name for the query. |
-| `input`        | Defines the means of event consumption via [streams](../source/source-types/stream-source.md), [named-windows](../windows/index.md), [tables](table-collection.md), and/or [named-aggregations](../aggregations/index.md), and defines the processing logic using [filters](#filter), [windows](#window), [stream-functions](#stream-function), [joins](#join), [patterns](#pattern) and [sequences](#sequence). |
+| `input`        | Defines the means of event consumption via [streams](../source/source-types/stream-source.md), [named-windows](../windows/index.md), [tables](../sink/sink-types/table/index.md), and/or [named-aggregations](../aggregations/index.md), and defines the processing logic using [filters](#filter), [windows](#window), [stream-functions](#stream-function), [joins](#join), [patterns](#pattern) and [sequences](#sequence). |
 | `projection`   | Generates output event attributes using [select](#select), [functions](#function), [aggregation-functions](#aggregation-function), and [group by](#group-by) operations, and filters the generated the output using [having](#having), [limit & offset](#limit-offset), [order by](#order-by), and [output rate limiting](#output-rate-limiting) operations before sending them out. Here the projection is optional and when it is omitted all the input events will be sent to the output as it is. |
```

Comment:
```suggestion
| `projection`   | (Optional) Generates output event attributes using [select](#select), [functions](#function), [aggregation-functions](#aggregation-function), and [group by](#group-by) operations, and filters the generated the output using [having](#having), [limit & offset](#limit-offset), [order by](#order-by), and [output rate limiting](#output-rate-limiting) operations before sending them out. If not provided, all the input events are sent to output as-is. |
```

Reply:
I am a fan of including "then" when you start a sentence with "if." It's a helpful signpost for less-skilled readers.